src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -116,7 +116,7 @@ def try_path_munging(
 
 
 def set_tags(scope: Scope, result: JSONData) -> None:
-    scope.set_tag("stacktrace_link.found", result.get("sourceUrl", False))
+    scope.set_tag("stacktrace_link.found", result["sourceUrl"] is not None)
```

Comment:
This is what [the original code](https://github.com/getsentry/sentry/pull/43105/files#diff-9ed2a929b59278cbb7d160ac361180e90dbb3ec7359f7543b614e12e1615d059L242-L243) looked like:
<img width="266" alt="image" src="https://user-images.githubusercontent.com/44410/212741646-b28c73a8-a5b1-4140-b0ab-f4b238841326.png">

You can see that `sourceUrl` is initialized with `None`:
https://github.com/getsentry/sentry/blob/947635eac3ae62ac7f7f566aa5b31350accfccfe/src/sentry/api/endpoints/project_stacktrace_link.py#L185